src/HealthChecks.System/FolderHealthCheckOptions.cs

```diff
@@ -0,0 +1,15 @@
+namespace HealthChecks.System
+{
+    /// <summary>
+    /// Options for <see cref="FolderHealthCheck"/>.
+    /// </summary>
+    public class FolderHealthCheckOptions
```

Comment:
Also note `CheckAllDrives` option from `DiskStorageOptions`. It makes sense to do the same here.